runtime/calamari/tests/xcm_mock/parachain.rs

```diff
@@ -694,14 +714,16 @@ pub(crate) fn create_asset_metadata(
     evm_address: Option<H160>,
     is_frozen: bool,
     is_sufficient: bool,
-) -> AssetRegistrarMetadata {
-    AssetRegistrarMetadata {
-        name: name.as_bytes().to_vec(),
-        symbol: symbol.as_bytes().to_vec(),
-        decimals,
+) -> AssetRegistryMetadata<Balance> {
+    AssetRegistryMetadata {
+        metadata: AssetStorageMetadata {
+            name: name.as_bytes().to_vec(),
+            symbol: symbol.as_bytes().to_vec(),
+            decimals,
+            is_frozen,
```

Comment:
could you explain your thoughts behind grouping some as metadata and some not?
e.g. why is `is_frozen` metadat and `is_sufficient` not?

Reply:
This is all that is required for the pallet-assets metadata https://github.com/paritytech/substrate/blob/master/frame/assets/src/types.rs#L127
There's a comment here  https://github.com/Manta-Network/Manta/blob/manta/primitives/manta/src/assets.rs#L103